### ISSUE TITLE
Master Template does NOT include parameters for DB Restore from Snapshot or Clone EFS

### ISSUE THREAD

**Issue by michael-newman** [awslabs/aws-refarch-wordpress#42]
Launching the stack via the "Launch Stack" button does NOT prompt for:

- Clone EFS, use the System ID of an existing EFS Filesystem.

- DBRestore from Snapshot, enter the cluster Snapshot name from the RDS console. (formatted rds:wordpress-stack-name-rds-xxxxxxxx-databasecluster-apzdbrozmzcn-snapshot-date)

However, the individual Templates (03-efsfilesystem, 03-rds) clearly have provisions regarding the same.

Additionally, the 2 links to review the [master] template gives a 404 error.

**Comment by michael-newman** [comment]
@darrylsosborne ... would welcome your thoughts!

**Comment by mteichtahl** [comment]
I had the same. I used a default of '' and this resolved the issue.

**Comment by badcrocodile** [comment]
I've been running into these very same issues @michael-newman, using the files provided here on GitHub. I have noticed that by using the S3 versions, everything builds as expected. Did you ever resolve your issues?

**Comment by michael-newman** [comment]
@badcrocodile, yes, resolved; but, I would not say eloquently in the CloudFormation templates. If I’m recalling correctly, there was a disconnect between the Master Template and the individual templates—so, we wound up running templates individually which got us past the RDS issue noted above; however, we had a different issue re EFS w/ VPC peering which is documented in issue #44.

**Comment by krishnu9** [comment]
@mteichtahl, where did you use a default of ""? In the master CloudFormation template or in the input parameters while deploying individual rds template?
